v2/stack_analyses.yaml

```diff
@@ -0,0 +1,623 @@
+openapi: 3.0.1
+info:
+  title: Stack Analyses API
+  version: "2.0"
+servers:
+  - url: https://f8a-analytics-2445582058137.{environment}.gw.apicast.io/api/v2
+    variables:
+      environment:
+        default: production # Production server
+        enum:
+          - production # Production server
+          - staging # Staging server
+
+security: # All endpoints needs 3scale user_key, refer securitySchemes
+  - tokenAuth: []
+
+paths:
+  /stack-analyses:
+    post:
+      tags:
+      - Scan Services
+      summary: |
+        Initiates a scan to gather security and other details about 
+        an application stack and related recommendations to improve 
+        the stack
+      parameters:
+      - name: uuid
+        in: header
+        description: Unique identifier for an User.
+        required: false
```

Comment:
Please make required as true with a default value as NA (similar to what we have in component analysis).
cc: @deepak1725